bindings/node.js/.gitignore

```diff
@@ -0,0 +1,3 @@
+build
+ckzg.node
```

Comment:
This is the NodeJS "native addon module" created by [node-gyp](https://github.com/nodejs/node-gyp).